src/components/pages/gfcrPages/GfcrIndicatorSetForm/modals/FinanceSolutionModal.js

```diff
@@ -65,8 +65,6 @@ const FinanceSolutionModal = ({
         id: financeSolution?.id,
         used_an_incubator:
           formikValues.used_an_incubator === 'none' ? null : formikValues.used_an_incubator,
-        local_enterprise: formikValues.local_enterprise === 'true',
-        gender_smart: formikValues.gender_smart === 'true',
```

Comment:
This should have worked in my mind, but it was also an unnecessary step. The changes solved the bug, and I am not sure why! (hence why the test steps are a little 'extra')